src/main/webapp/app/shared/notification/notification-popup/notification-popup.component.ts

```diff
@@ -68,12 +71,43 @@ export class NotificationPopupComponent implements OnInit {
         }
     }
 
+    // we use this method instead of the regular translation pipe to be able to also display legacy notifications that were created
+    // before it was possible to translate notifications
+    getNotificationTitleTranslation(notification: Notification): string {
+        const translation = this.artemisTranslatePipe.transform(notification.title);
+        if (translation?.includes(translationNotFoundMessage)) {
+            return notification.title ? notification.title : 'No title found';
```

Comment:
You could add a translation for the error messages as well. Additionally it can be written shorter:
```suggestion
            return notification.title ?? 'No title found';
```